parser/parser.go

```diff
@@ -162,14 +162,22 @@ func (p *Parser) Errors() []string {
 
 // returns Identifier AST node created from current token
 func (p *Parser) parseIdentifier() ast.Expression {
-	return &ast.Identifier{Token: p.curToken, Value: p.curToken.Literal}
+	ident := &ast.Identifier{Token: p.curToken, Value: p.curToken.Literal}
+
+	if p.peekTokenIs(token.ASSIGN) {
+		msg := fmt.Sprintf("cannot reassign constant: %q at line: %d", p.curToken.Literal, p.curToken.LineNumber)
+		p.errors = append(p.errors, msg)
+		p.nextToken() // TODO review pls
+	}
```

Comment:
adds error message when trying to do something like 
```go
const foo = "bar";
foo = "baz";
```

Reply:
but it just adds yet another error message and might make the output less readable instead